app/helpers/application_helper/button/miq_report_edit.rb

```diff
@@ -0,0 +1,13 @@
+class ApplicationHelper::Button::MiqReportEdit < ApplicationHelper::Button::Basic
+  include ApplicationHelper::Button::Mixins::XActiveTreeMixin
+
+  def visible?
+    reports_tree? ? custom_report_info? : true
```

Comment:
I don't see any definition for method `reports_tree?`, isn't something missing here?